mkosi/__init__.py

```diff
@@ -1638,7 +1638,7 @@ def run_selinux_relabel(state: MkosiState) -> None:
 
 
 def need_build_packages(config: MkosiConfig) -> bool:
-    return config.build_script is not None and len(config.build_packages) > 0
+    return bool(config.build_scripts) and len(config.build_packages) > 0
```

Comment:
Either use `len()` for both of these or `bool()` for both of these, but using both doesn't seem right

Reply:
I didn't look at this too closely when going through all the renames . I agree, it's doesn't seem right 😅.